pkgs/tools/networking/linux-router/default.nix

```diff
@@ -0,0 +1,89 @@
+# SOURCE: https://github.com/NixOS/nixpkgs/pull/92787/files
+
+{ stdenv, lib, fetchFromGitHub, makeWrapper
+
+# Dependencies
+, bash , procps , iproute2 , dnsmasq , iptables
+
+# WiFi hotspot dependencies
```

Comment:
Can you please place each of these optional packages below their respective `use*` (this is common practice).

Reply:
It's done, thanks :)